css/components/badges.css

```diff
@@ -1,99 +1,99 @@
 .badge-wrapper {
-	min-width: 2rem;
-	padding: 1rem;
-	font-size: 1.5rem;
+  min-width: 2rem;
+  padding: 1rem;
+  font-size: 1.5rem;
 }
 
 .badge {
-	position: relative;
+  position: relative;
 }
 
 .badge-counter {
-	position: absolute;
-	font-size: 0.8rem;
-	min-width: 1.5rem;
-	min-height: 1.5rem;
-	display: flex;
-	align-items: center;
-	justify-content: center;
+  position: absolute;
+  font-size: 0.8rem;
+  min-width: 1.5rem;
+  min-height: 1.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
same as above

Reply:
I have seprate utility classes in https://github.com/ttahm3d/chunks/blob/dev/css/components/layout.css. I don't want user to enter another class.